src/test/run-make/raw-dylib-stdcall-ordinal/output.txt

```diff
@@ -0,0 +1,2 @@
+exported_function_stdcall(6)
```

Comment:
Would you mind renaming this file to `expected_output.txt`? I think that would make the test easier to read.